transport-native-epoll/src/main/java/io/netty/channel/unix/Socket.java

```diff
@@ -428,4 +437,6 @@ private static native DatagramSocketAddress recvFromAddress(
     private static native void setSoLinger(int fd, int soLinger) throws IOException;
     private static native void setTcpDeferAccept(int fd, int deferAccept) throws IOException;
     private static native void setTcpQuickAck(int fd, int quickAck) throws IOException;
+
+    private static native PeerCredentials peerCredentials(int fd) throws IOException;
```

Comment:
Thanks for the explanation @normanmaurer. @cmdc0de - Lets stay consistent within this class and keep `get` prefix for getters.